src/SFA.DAS.ApprenticeFeedback.Jobs/Functions/ApprenticeFeedbackSummaryFunction.cs

```diff
@@ -2,34 +2,31 @@
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Extensions.Http;
 using Microsoft.Extensions.Logging;
-using NServiceBus;
-using SFA.DAS.ApprenticeFeedback.Jobs.Domain.Messages.Commands;
-using SFA.DAS.ApprenticeFeedback.Jobs.Infrastructure;
+using SFA.DAS.ApprenticeCommitments.Jobs.Api;
 using System;
+using System.Threading.Tasks;
 
 namespace SFA.DAS.ApprenticeFeedback.Jobs.Functions
 {
     public class ApprenticeFeedbackSummaryFunction
     {
         private readonly ILogger<GenerateFeedbackTransactionsFunction> _log;
-        private readonly IFunctionEndpoint _endpoint;
+        private readonly IApprenticeFeedbackApi _api;
 
-        public ApprenticeFeedbackSummaryFunction(ILogger<GenerateFeedbackTransactionsFunction> log, IFunctionEndpoint endpoint)
+        public ApprenticeFeedbackSummaryFunction(ILogger<GenerateFeedbackTransactionsFunction> log, IApprenticeFeedbackApi api)
         {
             _log = log;
-            _endpoint = endpoint;
+            _api = api;
```

Comment:
@ChrisJWoodcock  can you rename variable `api `to `apprenticeFeedbackApi` as its more descriptive

Reply:
Done and lots of similar ones